code-challenges.js

```diff
@@ -13,8 +13,29 @@ var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
-
-
+// Create a function that takes in one argument of temperature
+const isBoiling = (temp) => {
```

Comment:
awesome function naming! However, generally a function name starts with `is` should return `true` or `false`. Still good to maintain this practice of function naming. 😸